runelite-client/src/main/java/net/runelite/client/plugins/freezetimers/Timers.java

```diff
@@ -101,14 +98,20 @@ public long getTimerReApply(Actor actor, TimerType type)
 
 	public boolean areAllTimersZero(Actor actor)
 	{
-		for (TimerType type : TimerType.values())
+		Iterator typeIterator = timerMap.keySet().iterator();
```

Comment:
final

Reply:
Could also just make it a Iterator<TimerType>, so you don't have to do (TimerType)